.github/workflows/pr.yml

```diff
@@ -277,11 +277,11 @@ jobs:
           if [ "$numPass" -eq 0 ];then echo "*** 0 Passed! ***";exit 2;fi
           if [ "$numPass" -lt 33 ];then echo "*** Failed to pass at least 33 ! ***";exit 2;fi
           if [ "$numPass" -eq 0 ];then echo "*** Passed! ***";exit 0;fi
-  functional_docker_windows:
+  functional_scheduled_stop_docker_windows:
```

Comment:
lets add this as its own job, since test functional itself is flaky

also needs to be added to both PR and master.yml

Reply:
thanks